src/Microsoft.DotNet.ProjectModel/FileFormatException.cs

```diff
@@ -24,7 +24,7 @@ public sealed class FileFormatException : Exception
         
         public override string ToString()
         {            
-            return $"{Path}({Line},{Column}): Error: {base.ToString()}";
+            return string.Format("{0}({1},{2}): {3}", Path, Line, Column, base.Message);
```

Comment:
I'm confused, too. But this actually fixes the exception thrown from Console.Logger. I'm all for fix issue over there. But the intention here is unblock our VS partner. #1569